pkgs/utkuozdemir/pv-migrate/registry.yaml

```diff
@@ -0,0 +1,50 @@
+packages:
+  - type: github_release
+    repo_owner: utkuozdemir
+    repo_name: pv-migrate
+    description: CLI tool to easily migrate Kubernetes persistent volumes
+    asset: pv-migrate_{{.Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+    format: tar.gz
+    overrides:
+      - goos: windows
+        format: zip
+    replacements:
+      amd64: x86_64
+    supported_envs:
+      - darwin
+      - linux
+      - amd64
+    checksum:
+      type: github_release
+      asset: checksums.txt
+      algorithm: sha256
+    version_constraint: semver(">= 0.8.1")
+    version_overrides:
+      - version_constraint: semver(">= 0.6.3")
+        asset: pv-migrate_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+      - version_constraint: semver(">= 0.5.5")
+        asset: pv-migrate_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+        rosetta2: true
+      - version_constraint: semver(">= 0.5.4")
+        asset: pv-migrate_{{trimV .Version}}_{{.OS}}_{{.Arch}}.{{.Format}}
+        supported_envs:
+          - darwin
+          - amd64
+        rosetta2: true
+      - version_constraint: semver(">= 0.2.1")
+        supported_envs:
+          - darwin
+          - amd64
+        rosetta2: true
+      - version_constraint: semver("< 0.2.1")
+        asset: pv-migrate-{{.OS}}.{{.Format}}
+        overrides: []
+        replacements: {}
+        supported_envs:
+          - darwin
+          - amd64
+        rosetta2: true
+        checksum:
+          enabled: false
+        files:
+          - src: bin/{{.OS}}/pv-migrate
```

Comment:
I guess `name` is mandatory.

Reply:
fixed https://github.com/aquaproj/aqua-registry/pull/12352/commits/62ace6da0acc2d286ecc58df1075104fb962054c